src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
What needs this package? `Microsoft.Extensions.Logging.Abstractions` should be enough.